### PR TITLE
Update how-to-catch-outgoing-emails-locally.md

### PR DESCRIPTION
**Pull request by moi238**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: docs/how-to-catch-outgoing-emails-locally.md [hunk 1]
```diff
@@ -34,9 +34,9 @@ When the installation completes, you can start [using MailHog](#using-mailhog).
 
 ### Installing MailHog on Windows
 
-Download the latest version of MailHog from [MailHog's official repository](https://github.com/mailhog/MailHog/releases). Locate and click on the link for your Windows version (32 or 64 bit) and .exe file will be downloaded to your computer.
+Download the latest version of MailHog from [MailHog's official repository](https://github.com/mailhog/MailHog/releases). Locate and click on the link for your Windows version (32 or 64 bit) and a .exe file will be downloaded to your computer.
 
-When the download completes, click to open the file. A Windows firewall notification may appear requesting access permission for MailHog. A standard Windows command line prompt will open where MailHog will be running once firewall access is granted.
+When the download completes, click to open the file. A Windows firewall notification may appear, requesting access permission for MailHog. A standard Windows command line prompt will open, where MailHog will be running once firewall access is granted.
```
Comment (RandellDawson) [inline docs/how-to-catch-outgoing-emails-locally.md]: Please revert the changes on this line.

File: docs/how-to-catch-outgoing-emails-locally.md [hunk 2]
```diff
@@ -46,13 +46,13 @@ Start [using MailHog](#using-mailhog).
...
 
-Run the following commands to install GO on Debian-based systems like Ubuntu and Linux Mint.
+Run the following commands to install GO on Debian-based systems, like Ubuntu and Linux Mint.
 
 ```bash
 sudo apt-get install golang
 ```
 
-Run the following commands to install GO on RPM-based systems like CentOS, Fedora, Red Hat Linux, ect.
+Run the following commands to install GO on RPM-based systems, like CentOS, Fedora, Red Hat Linux, etc.
```
Comment (RandellDawson) [inline docs/how-to-catch-outgoing-emails-locally.md]: Remove the comma:
```suggestion
Run the following commands to install GO on RPM-based systems like CentOS, Fedora, Red Hat Linux, etc.
```

File: docs/how-to-catch-outgoing-emails-locally.md [hunk 3]
```diff
@@ -64,7 +64,7 @@ Alternatively, run the following commands to install GO.
 sudo yum install golang
 ```
 
-Now set the path for Go with the following commands.
+Now set the path for Go, with the following commands.
```
Comment (RandellDawson) [inline docs/how-to-catch-outgoing-emails-locally.md]: Revert the changes to this line.

File: docs/how-to-catch-outgoing-emails-locally.md [hunk 4]
```diff
@@ -84,15 +84,15 @@ Start [using MailHog](#using-mailhog).
 
 ## Using MailHog
 
-Open a new browser tab or window and navigate to [http://localhost:8025](http://localhost:8025) to open your MailHog inbox when the MailHog installation has completed and MailHog is running. The inbox will appear similar to the screen shot below.
+Open a new browser tab, or window, and navigate to [http://localhost:8025](http://localhost:8025) to open your MailHog inbox when the MailHog installation has completed, and MailHog is running. The inbox will appear similar to the screen shot below.
```
Comment (RandellDawson) [inline docs/how-to-catch-outgoing-emails-locally.md]: Revert the changes to this line.

File: docs/how-to-catch-outgoing-emails-locally.md [hunk 5]
```diff
@@ -84,15 +84,15 @@ Start [using MailHog](#using-mailhog).
...
 
 ![MailHog Screenshot 1](images/mailhog/1.jpg)
 
 Emails sent by your freeCodeCamp installation will appear as below
 
 ![MailHog Screenshot 2](images/mailhog/2.jpg)
 
-Two tabs that allow you to view either plain text or source content will be available when you open a given email. Ensure that the plain text tab is selected as below.
+Two tabs that allow you to view either plain text, or source content, will be available when you open a given email. Ensure that the plain text tab is selected as below.
```
Comment (RandellDawson) [inline docs/how-to-catch-outgoing-emails-locally.md]: Revert changes to this line.

File: docs/how-to-catch-outgoing-emails-locally.md [hunk 6]
```diff
@@ -46,13 +46,13 @@ Start [using MailHog](#using-mailhog).
 
 First, install [Go](https://golang.org).
 
-Run the following commands to install GO on Debian-based systems like Ubuntu and Linux Mint.
+Run the following commands to install GO on Debian-based systems, like Ubuntu and Linux Mint.
```
Comment (RandellDawson) [inline docs/how-to-catch-outgoing-emails-locally.md]: Please revert the changes to this line.

**Review by RandellDawson — CHANGES_REQUESTED** [review]
I have requested several changes.

Thank you.

**Review by moi238 — COMMENTED** [review]
All changes made as requested.

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Review by RandellDawson — APPROVED** [review]

**Review by RandellDawson — APPROVED** [review]

**Pull request merged**